include/aws/common/posix/common.inl

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#ifndef AWS_COMMON_POSIX_COMMON_INL
+#define AWS_COMMON_POSIX_COMMON_INL
+
+#include <aws/common/common.h>
+
+#include <errno.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static inline int aws_private_convert_and_raise_error_code(int error_code) {
```

Comment:
Just curious, why isn't this one `AWS_STATIC_IMPL`?

Reply:
This is not a public API, so we don't want to export symbols for it.

Reply:
Ah, that makes sense.